tfx/components/tuner/executor_test.py

```diff
@@ -127,6 +127,26 @@ def testTuneArgs(self):
           output_dict=self._output_dict,
           exec_properties=self._exec_properties)
 
+  def testDoWithCustomSplits(self):
```

Comment:
Shouldn't this test with actual custom splits (different from "train" and "eval")? Otherwise test results would be no different if the code actually ignores "splits" entirely.

Reply:
I updated the tests for custom splits. Can you please check if it's good now?

Reply:
It's still not clear to me how we test that the custom splits are being used. If tuner executor ignored 'training' and 'evaluating' splits and instead used 'train' and 'eval' splits, would this test fail?

Reply:
basically the split here should match the input example artifact, the test example only contains `train` and `eval` now.

This function is more like testDoWithSpecifiedSplits to make sure the user can specify the split by config (thus custom split names can work if examplegen generates custom splits).

Reply:
Let me rephrase my question: If the code being tested completely ignored TrainArgs.splits field, so instead of "training" and "evaluating" splits it used the default "train" and "eval" splits, will this test fail?

Reply:
line138 changed the split name of example artifact, if train and eval is used, it will fail.

Tina, we can also remove train and eval data to make sure

Reply:
thanks for confirming

Reply:
it's updated now, i created a new example artifact so in the data folder there's only 'training' and 'evaluating', if it used 'train' and 'eval', it will fail.